model/status_board/v1/product_resource.model

```diff
@@ -27,10 +27,12 @@ resource Product {
   method Delete {
   }
 
+  // /api/status-board/v1/products/{id}/applications
```

Comment:
I think it is redundant to have this here, as this is automatically calculated by the model and added to the generated OpenAPI documentation. If you really want to include it then add a first sentence explaining that this is, and make this a complete example:

```
// Reference to the service that manages the collection
// of applications associated to this product. For example
// go get the list of applications for product `123` send a
// request like this:
//
// ```http
// GET /api/status-board/v1/products/123/applications
// ```
```